tests/drum/unit/test_typeschema_validation.py

```diff
@@ -140,7 +140,7 @@ def missing_data(self, data):
 
     @pytest.fixture
     def sparse_df(self):
-        yield pd.DataFrame.sparse.from_spmatrix(sparse.eye(10))
+        yield sparse.csr_matrix(sparse.eye(10))
```

Comment:
Why was this change necessary?

Reply:
to correctly match what is received when sparse data is returned from a transform.